clang/lib/Tooling/Inclusions/Stdlib/CSymbolMap.inc

```diff
@@ -475,6 +475,12 @@ SYMBOL(fmaxl, None, <math.h>)
 SYMBOL(fmin, None, <math.h>)
 SYMBOL(fminf, None, <math.h>)
 SYMBOL(fminl, None, <math.h>)
+SYMBOL(fmaximum_num, None, <math.h>)
```

Comment:
Not sure what this for, but this isn't tested?

Reply:
Oh, yes. It is autogenerated from https://github.com/PeterFeicht/cppreference-doc/releases.
If we need to update it, we should update cppreference.